samples/ClientSample/ClientSample.csproj

```diff
@@ -1,11 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Yeah, but I was curious what the difference between these Skds was - clearly a console app can use .Web SDK just fine...